extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -300,11 +300,11 @@ public function unblock() {
 			return;
 		}
 
-		$this->response->setFormat('json');
-		$this->setVal('success', false);
+		$this->response->setFormat( 'json' );
+		$this->setVal( 'success', false );
 
 		if ( !$this->userCanExecute( $this->wg->User ) ) {
-			$this->setVal('error', 'permission');
+			$this->setVal( 'error', 'permission' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)